docs/release-notes/8.7.asciidoc

```diff
@@ -1,4 +1,37 @@
 [[ecs-release-notes-8.7.0]]
 === 8.7.0
 
-coming[8.7.0]
+[[schema-changes-8.7.0]]
+[float]
+==== Schema changes
+
+[[schema-bugfixes-8.7.0]]
+[float]
+===== Bugfixes
+
+* Remove duplicated `client.domain` definition #2120
```

Comment:
For all the PR links, you'll want to format like `{ecs_pull}2010[#2120]` so that it links properly in the [docs](https://www.elastic.co/guide/en/ecs/current/ecs-release-notes-8.6.1.html)